.workbench/manifest.json

```diff
@@ -0,0 +1,197 @@
+{
+    "version": "1.0.0",
+    "toolchains": [
+        {
+            "firmware": "deviceOS@source",
+            "compilers": "gcc-arm@5.3.1",
+            "debuggers": "openocd@0.11.2-adhoc6ea4372.0",
+            "platforms": [6, 8, 10, 12, 13, 14, 23],
+            "scripts": "buildscripts@1.7.1",
```

Comment:
this would ideally be `buildscripts@1.8.0`

Reply:
:man_facepalming: whaaat....

Presto, change-o! Behold the magic of `--amend`!
:tophat: :rabbit: